File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/metrics/audio/UnknownAudioType.java
package com.mapbox.services.android.navigation.v5.navigation.metrics.audio;

import android.content.Context;

public class UnknownAudioType implements AudioTypeResolver {
  private static final String UNKNOWN = "unknown";

  @Override
  public void nextChain(AudioTypeResolver chain) {
  }

  @Override
  public String obtainAudioType(Context context) {
    return UNKNOWN;
  }
}

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/metrics/audio/UnknownAudioType.kt
package com.mapbox.services.android.navigation.v5.navigation.metrics.audio

import android.content.Context

internal class UnknownAudioType : AudioTypeResolver {
    private val UNKNOWN = "unknown"

    override fun nextChain(chain: AudioTypeResolver) {}

    override fun obtainAudioType(context: Context): String {
        return UNKNOWN
    }

}
